src/main/resources/data/modern_industrialization/recipes/vanilla_recipes/assembler/composter.json

```diff
@@ -0,0 +1,23 @@
+{
+    "type": "modern_industrialization:assembler",
```

Comment:
Moving this one to the packer to avoid conflicting with the barrel

Reply:
I'm adding seeds to the composter recipe, because only moving to the packer creates a conflict with the pressure plate recipe.